### PR TITLE
Plot and print features

### PR DESCRIPTION
**Pull request by priyakasimbeg**
- added arguments for passing down use_bias and kernel_constraint from FNN to tf.layers.Dense.
- added __str__ in model to easily get string representation of model info like weights and biases (useful for saving to file).
- addes save options for loss plots and plot from best state

### CODE REVIEW

File: deepxde/maps/fnn.py [hunk 1]
```diff
@@ -56,7 +60,7 @@ def build(self):
             y = self._input_transform(y)
         for i in range(len(self.layer_size) - 2):
             if self.batch_normalization is None:
-                y = self.dense(y, self.layer_size[i + 1], activation=self.activation)
+                y = self.dense(y, self.layer_size[i + 1], activation=self.activation, use_bias=self.use_bias)
```
Comment (lululxvi) [inline deepxde/maps/fnn.py:63]: Should also modify line 72

File: deepxde/model.py [hunk 2]
```diff
@@ -43,6 +43,14 @@ def __init__(self, data, net):
         self.stop_training = False
         self.callbacks = None
 
+    def __str__(self):
```
Comment (lululxvi) [inline deepxde/model.py:46]: This is not very necessary. There is a `state_dict()` function to return a dict.
